Power_Protection.lib

```diff
@@ -192,7 +192,6 @@ F2 "Package_TO_SOT_SMD:SOT-363_SC-70-6" 0 -450 50 H I C CNN
 F3 "" 80 75 50 H I C CNN
 $FPLIST
  *SC*70*
- *SC*88*
```

Comment:
It seems that NUP2202 is also available in SC-88 packaging.
Datasheet: https://www.onsemi.com/pub/Collateral/NUP2202W1-D.PDF
Maybe this component should have multiple filters but no default packaging. See KLC S5.1 "For generic symbols (those which map to multiple possible footprints), the Footprint field must be left blank".

What do you think?

Reply:
On [Datasheet](https://www.onsemi.com/pub/Collateral/NUP2202W1-D.PDF) and (Package Size Comparison)[http://www.onsemi.com/site/cdroms/micropackages/pdf-docs/onsemi/SOT563%20footprint-prof%23B3380.pdf]  SC-77, SC-88, SOT363, US6, UMT6, S-Mini6 have same dimensions.

What do you think about to represent this in footprint library and rename footprint? @antoniovazquezblanco

Reply:
I am learning new things here :)

For that I would like to get the input of someone with more experience than me. Let's ping someone and get some feedback:
@poeschlr, @SchrodingersGat, @pointhi, @evanshultz ?

Reply:
Learning new thinks :) this why I like to be here. 6 eyes see better than 4, get this checked by others cannot harm.

Reply:
Datasheet Page 1:
![image](https://user-images.githubusercontent.com/19774902/45752537-199fb700-bc16-11e8-9ea3-b6d694bd80ae.png)

Datasheet Page 8:
![image](https://user-images.githubusercontent.com/19774902/45752610-5f5c7f80-bc16-11e8-8c93-7ac13db71bba.png)

Reply:
Thank you for teaching me this stuff. You are totally right :).